devtools/startup/DevToolsStartup.jsm

```diff
@@ -1444,16 +1447,13 @@ function createRecordingButton() {
 
       const { target: node } = evt;
       const { gBrowser } = node.ownerDocument.defaultView;
-      const recording = gBrowser.selectedBrowser.hasAttribute(
-        "recordExecution"
```

Comment:
I suspect I still need to include this check but I wasn't sure. Rather than blindly include it, I wanted to confirm with others.

Reply:
Good question, I don't like this approach so I'll be happy to be rid of it. I think this mostly depends on my other question about `browser` being persistent if you drag the tab into a new window. I'm pretty sure it's not, so we'd have to find some other key, else look up the active recording based on some other key.